<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCsvFileDialog</class>
 <widget class="QDialog" name="ImportCsvFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import CSV File</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="fileNameLabel">
          <property name="text">
           <string>File name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="fileNameButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="delimiterLabel">
          <property name="text">
           <string>Delimiter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="delimiterComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Comma (&quot;,&quot;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Colon (&quot;:&quot;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bar (&quot;|&quot;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tab (&quot;    &quot;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Space (&quot; &quot;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Custom Delimiter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="customDelimiterLineEdit">
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Column Mapping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="mappingTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
